Documentation/ceph-block.md

```diff
@@ -12,22 +12,23 @@ indent: true
 {% endif %}
 # Block Storage
 
-Block storage allows you to mount storage to a single pod. This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
+Block storage allows a single pod to mount storage. This guide shows how to create a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
 
 ## Prerequisites
 
-This guide assumes you have created a Rook cluster as explained in the main [Quickstart](ceph-quickstart.md) guide.
+This guide assumes a Rook cluster as explained in the [Quickstart](ceph-quickstart.md).
 
 ## Provision Storage
 
-Before Rook can start provisioning storage, a StorageClass and its storage pool need to be created. This is needed for Kubernetes to interoperate with Rook for provisioning persistent volumes. For more options on pools, see the documentation on [creating storage pools](ceph-pool-crd.md).
+Before Rook can provision storage, a [`StorageClass`](https://v1-13.docs.kubernetes.io/docs/concepts/storage/storage-classes) and [`CephBlockPool`](ceph-pool-crd.md) need to be created. This will allow Kubernetes to interoperate with Rook when provisioning persistent volumes.
```

Comment:
This link goes to 1.13, which has a warning that it's no longer actively maintained. How about a more recent link such as: `https://kubernetes.io/docs/concepts/storage/storage-classes/`

Reply:
Yes, I was trying to follow the pattern as per the other link you reference, but pointing to the latest doc makes sense.